nomos-services/cryptarchia-consensus/src/leadership.rs

```diff
@@ -3,6 +3,7 @@ use cryptarchia_ledger::{Coin, Commitment, Config, EpochState, LeaderProof};
 use nomos_core::header::HeaderId;
 use std::collections::HashMap;
 
+#[derive(Clone, Debug)]
```

Comment:
because [`proptest::StateMachineTest`](https://docs.rs/proptest-state-machine/latest/proptest_state_machine/test_runner/trait.StateMachineTest.html) requires these.